MARS_Specflow/Hooks/TestHooks.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace MARS.Hooks
+{
+    [Binding]
+    public class TestHooks
+    {
+        public static IWebDriver driver; // Shared WebDriver instance
+
+        [BeforeScenario]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+        }
+
+        [AfterScenario]
+        public void TearDown()
```

Comment:
you should also add hooks for cleanup

Reply:
more information on hooks
Please see
https://codebun.com/how-to-implement-hooks-in-specflow-c/